### PR TITLE
added extra presence

### PR DESCRIPTION
**Pull request by mikytron123**

### CODE REVIEW

File: NadekoBot.Core/Services/NadekoBot.cs [hunk 1]
```diff
@@ -94,7 +94,8 @@ public NadekoBot(int shardId, int parentProcessId)
                 ConnectionTimeout = int.MaxValue,
                 TotalShards = Credentials.TotalShards,
                 ShardId = shardId,
-                AlwaysDownloadUsers = false
+                AlwaysDownloadUsers = false,
+                GatewayIntents = GatewayIntents.All
```
Comment (Gibstick) [inline NadekoBot.Core/Services/NadekoBot.cs:98]: You should add just the one intent or else the bot will get flooded with messages it doesn't need to handle

**Review by Gibstick — CHANGES_REQUESTED** [review]

File: NadekoBot.Core/Services/NadekoBot.cs (hunk 1 above)
Comment (mikytron123) [reply NadekoBot.Core/Services/NadekoBot.cs:98]: The bot always had every single unprivileged intent by default. This just changes it to include 2 extra ones

Comment (mikytron123) [reply NadekoBot.Core/Services/NadekoBot.cs:98]: https://discordnet.dev/api/Discord.WebSocket.DiscordSocketConfig.html read here the default is allunprivileged

Comment (mikytron123) [reply NadekoBot.Core/Services/NadekoBot.cs:98]: The bot doesn't get flooded with messages it doesn't need to handle, it uses most of the intents

Comment (Gibstick) [reply NadekoBot.Core/Services/NadekoBot.cs:98]: ok fine, it won't be much worse but there are a ton of features we don't use. But trying to whittle down the intents would take too long

**Review by Gibstick — APPROVED** [review]

**Pull request merged**